### PR TITLE
Bug 1219564 - add mouse cursor in bookmark and history

### PR DESCRIPTION
**Pull request by danhuang1202**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=95c3fa2d91628076fceb660157f9bbeed46845cd

File: tv_apps/browser/js/awesomescreen.js [hunk 1]
```diff
@@ -3166,5 +3068,81 @@ var Awesomescreen = {
...
       Awesomescreen.pointerImg.style.display = 'none';
     }
+  },
+
+  /**
+   * Handle smartList event binding
+   * @param {Node} el - Dom element of smartList.
+   * @param {Object} list  - SmartList object.
+   * @param {Object} store - Store object.
+   * @param {String} list  - SmartList display type, now support.
```
Comment (begeeben) [inline tv_apps/browser/js/awesomescreen.js]: ```type```, typing error.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fa481452c8a2f76eae15731dc87b1fd3fb73d2d3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7cb1d67a1968b106086b9283d49b0d45c769a744

**Pull request merged**